wayland-server/src/global.rs

```diff
@@ -135,9 +135,9 @@ pub trait DelegateGlobalDispatch<
 
 #[macro_export]
 macro_rules! delegate_global_dispatch {
-    ($dispatch_from: ty: [$($interface: ty),*] => $dispatch_to: ty) => {
+    ($dispatch_from:ident $(< $( $lt:tt $( : $clt:tt $(+ $dlt:tt )* )? ),+ >)? : $interface: ty => $dispatch_to: ty) => {
         $(
```

Comment:
You kept the outer expansion on this macro

Reply:
:woman_facepalming:

I added a smoke test to actually compile the expanded code from `delegate_global_dispatch` once.